### PR TITLE
Add clarification for reolink entities

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add clarification for reolink entities

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/87932
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by starkillerOG** [comment]
@frenck can you merge this?
(It is about the core PRs you merged).

**Comment by frenck** [comment]
@starkillerOG Can you please stop mentioning and pulling people into PRs?

https://developers.home-assistant.io/docs/review-process#what-not-to-do

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -63,6 +63,10 @@ Depending on the supported features of the camera number entities are added for:
 - AI vehicle sensitivity
 - AI pet sensitivity
 
+`Floodlight turn on brightness` controlles the brightness of the floodlight when it is turned on internally by the camera (see `select.floodlight_mode` entity) or when using the `light.floodlight` entity.
```
Comment (frenck) [inline source/_integrations/reolink.markdown]:
```suggestion
"Floodlight turn on brightness" controls the brightness of the floodlight when it is turned on internally by the camera (see "Floodlight mode" select entity) or when using the "Floodlight" light entity.
```

File: source/_integrations/reolink.markdown [hunk 2]
```diff
@@ -63,6 +63,10 @@ Depending on the supported features of the camera number entities are added for:
...
+
+When the camera is not moved and no person/pet/vehicle is detected for the `Guard return time` in seconds, and the `switch.guard_return` is ON, the camera will move back to the guard position.
```
Comment (frenck) [inline source/_integrations/reolink.markdown]:
```suggestion
When the camera is not moved and no person/pet/vehicle is detected for the "Guard return time" in seconds, and the "Guard return" switch is ON, the camera will move back to the guard position.
```

File: source/_integrations/reolink.markdown [hunk 3]
```diff
@@ -76,6 +80,10 @@ Depending on the supported features of the camera button entities are added for:
 - Guard go to
 - Guard set current position
 
+PTZ left, right, up and down will continually move the camera in the respective position until the PTZ stop is called or the hardware limit is reached.
+
+`Guard set current position` will set the current position as the new guard position.
```
Comment (frenck) [inline source/_integrations/reolink.markdown]:
```suggestion
"Guard set current position" will set the current position as the new guard position.
```

File: source/_integrations/reolink.markdown [hunk 4]
```diff
@@ -119,9 +127,9 @@ Depending on the supported features of the camera light entities are added for:
 - Infra red lights in night mode
 - Status LED
 
-When the floodlight entity is ON always ON, when OFF controlled based on the internal camera floodlight mode (Off, Auto, Schedule)
+When the floodlight entity is ON always ON, when OFF controlled based on the internal camera floodlight mode (Off, Auto, Schedule), see `select.floodlight_mode` entity.
```
Comment (frenck) [inline source/_integrations/reolink.markdown]:
```suggestion
When the floodlight entity is ON always ON, when OFF controlled based on the internal camera floodlight mode (Off, Auto, Schedule), see the "Floodlight mode" select entity.
```

File: source/_integrations/reolink.markdown [hunk 5]
```diff
@@ -119,9 +127,9 @@ Depending on the supported features of the camera light entities are added for:
...
 
-When IR light entity is OFF always OFF, when ON IR LEDs will be on when camera is in night vision mode, see "Day night mode"
+When IR light entity is OFF always OFF, when ON IR LEDs will be on when camera is in night vision mode, see `select.day_night_mode" entity.
```
Comment (frenck) [inline source/_integrations/reolink.markdown]:
```suggestion
When IR light entity is OFF always OFF, when ON IR LEDs will be on when the camera is in night vision mode, see the "Day night mode" select entity.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @starkillerOG 👍 

../Frenck

**Pull request merged**